src/style/common.less

```diff
@@ -23,3 +23,4 @@
 
 @mobileWidth: 768px;
 @tabletWidth: 1092px;
+@desktopWidth: 1200px;
```

Comment:
I've made a mistake in a previous review and suggested a `@desktopWidth` but quickly corrected my self. My bad, I'm sorry if I created a confusion. 😔

I think we should go with two separators only as that leaves us with three layouts to design for which is more than enough. Please remove this and replace `@desktopWidth` with `@tabletWith` in other files.